wasm/app/index.js

```diff
@@ -1,26 +1,25 @@
-import * as rp from "rustpython_wasm";
+import * as rp from 'rustpython_wasm';
+
+// so people can play around with it
+window.rp = rp;
```

Comment:
Does this mean we can have an interactive repl in which variables are retained through calls to the eval function?

Reply:
I don't think so, because the functions make a new VM on every call, but my first instinct after opening the website and seeing the fibonacci demo was to open up the devtools and try to see what kind of interop there was with JS. With the module exposed, you could do something like:
```js
const pyOutput = rp.eval_py(`
import random
if random.random() < .5:
    return True
else:
    return False
`)
```

Reply:
Ah, this is nice! I think we will soon find a way to store the repl scope in some clever way.